util/nats-server-hardened.service

```diff
@@ -9,6 +9,9 @@ ExecReload=/bin/kill -s HUP $MAINPID
 ExecStop=/bin/kill -s SIGINT $MAINPID
 User=nats
 Group=nats
+# The nats-server uses SIGUSR2 to trigger using Lame Duck Mode (LDM) shutdown
+KillSignal=SIGUSR2
+# You might want to adjust TimeoutStopSec too.
```

Comment:
Hello, sorry for the late message, I only saw the change when browsing the release notes.

Shouldn't the ExecStop command also be updated to send SIGUSR2 ?